portal/src/store/modules/user.js

```diff
@@ -1,10 +1,8 @@
 import _ from 'lodash';
 import injector from 'vue-inject';
 
-
```

Comment:
Ik doe altijd twee regels tussen imports en constants en code. Python PEP8 regel, die ik ook op JS toepas.

Reply:
Ik vind het prima hoor. Ik trek het later wel recht met eslint, ik zie toch dat het nog niet consistent is doorgevoerd.